.travis.yml

```diff
@@ -26,18 +22,20 @@ before_install:
     source /etc/fsl/fsl.sh;
     source /etc/afni/afni.sh;
     export FSLOUTPUTTYPE=NIFTI_GZ; fi }
-- travis_retry bef_inst
-install:
-#  Add install of vtk and mayavi to test mesh (disabled): conda install -y vtk mayavi &&
-- function inst {
+- function conda_inst {
+  export CONDA_HOME=/home/travis/anaconda &&
```

Comment:
@satra I was just checking if switching to anaconda made any difference as regards the segmentation fault, but I'm arriving to the conclusion that miniconda is much better for travis. So there's no point on switching.